hadoop-hdds/framework/pom.xml

```diff
@@ -170,6 +170,12 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <groupId>org.awaitility</groupId>
       <artifactId>awaitility</artifactId>
     </dependency>
+    <dependency>
```

Comment:
I was doing this since `ManagedSecretKey.java` required `OzoneManagerProtocolProtos` from the package `org.apache.hadoop.ozone.protocol.proto` - I will take a look at this to see how this can be avoided, thanks.

Reply:
I have made changes to avoid importing this dependency.